source/slang/slang-ir.cpp

```diff
@@ -7869,6 +7869,21 @@ namespace Slang
         return type;
     }
 
+    IRType* IRInst::getUnratedAndUnattributedType()
```

Comment:
there is already a `unwrapAttributedType` defined in `slang-ir-util.h` that does exactly this.

Reply:
ah, I missed that searching for IRType->IRType functions, thanks